docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
+---
+
+# otelcol.exporter.loadbalancing
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`otelcol.exporter.loadbalancing` accepts logs and/or traces from other `otelcol` components
+and writes them over the network using the OpenTelemetry Protocol (OTLP) protocol. 
+
+> **NOTE**: `otelcol.exporter.loadbalancing` is a wrapper over the upstream
+> OpenTelemetry Collector `loadbalancing` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.loadbalancing` components can be specified by giving them
+different labels.
+
+Note that either a trace ID or a service name is used for the decision on which backend to use. 
+The actual backend load isn't taken into consideration. Even though this load-balancer won't do 
+round-robin balancing of the batches, the load distribution should be very similar among backends 
+with a standard deviation under 5% at the current configuration.
+
+`otelcol.exporter.loadbalancing` is especially useful for backends configured with tail-based samplers
+which make a decision based on the view of the full trace.
+
+When a list of backends is updated, around `1/n` of the space will be changed, so that the same trace ID 
```

Comment:
Agree with Clayton here, I am not sure what this means. What's a space? I think I can infer what it means but more text here would be nice.

Reply:
I agree that it is confusing. I just copy pasted it from the original Otel docs because tbh I'm not entirely sure about their math. I rephrased it differently, in a way that makes more sense to me, and if you think it makes sense then I'll also open a PR upstream to change the Otel docs. I changed their formula a little bit too.